src/utils/wallet.js

```diff
@@ -438,53 +647,266 @@ class Wallet {
         }
     }
 
-    async signatureFor(accountId) {
-        const blockNumber = String((await this.connection.provider.status()).sync_info.latest_block_height);
-        const signed = await this.inMemorySigner.signMessage(Buffer.from(blockNumber), accountId, NETWORK_ID);
+    async signatureFor(account) {
```

Comment:
not sure why this method needs changes

Reply:
@mattlockyer not sure I understand why anything would be signed before account is created? How will it be validated given there is no account (and so no keys yet)?